topics/authentication.md

```diff
@@ -6,140 +6,151 @@ Code examples: <a href="samples.md" anchor="authentication">Authentication</a>
 </p>
 </microformat>
 
-<include src="lib.xml" include-id="outdated_warning"/>
+Ktor provides the `Authentication` feature to handle authentication and access control. Typical usage scenarios include logging in users, granting access to specific resources, and secure transmitting information between parties. You can also use `Authentication` with [Sessions](sessions.md) to keep a user's information between requests.
 
+## Supported authentication types {id="supported"}
+Ktor supports different authentication schemes: from general HTTP authentication schemes such as Basic and Digest to OAuth 2.0 for accessing external resources.
 
-Ktor supports authentication out of the box. It supports mechanisms to read *credentials*, and to authenticate *principals*.
+### HTTP authentication {id="http-auth"}
+HTTP provides a [general framework](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication) for access control and authentication. In Ktor, you can use the following HTTP authentication schemes:
+* [Basic](basic.md) - an authentication method to provide a user name and password encoded using `Base64`.
+* [Digest](digest.md) - an authentication method that communicates user credentials in an encrypted form by applying a hash function to the username and password.
+* `Bearer` - an authentication scheme that involves security tokens called bearer tokens. You can use [JSON Web Tokens](#jwt) as bearer tokens and use the `jwt` authentication in Ktor to verify a token and validate the claims contained within it.
```

Comment:
You're referencing the word claims without explaining it. I'd just drop it here and say 

"in Ktor to validate and authorize a request" 

Then they can click on that topic to learn more.